src/constants.js

```diff
@@ -34,6 +40,29 @@ export const DIMENSIONS_LIST = [
   { id: DIMENSION_KEYS.age, label: DIMENSIONS.age },
 ];
 
+export const AGES = {
+  "<25": "<25",
+  "25-29": "25-29",
+  "30-34": "30-34",
+  "35-39": "35-39",
+  "40<": "40<",
+};
+
+export const GENDERS = {
+  FEMALE: "Female",
+  MALE: "Male",
+};
+
+export const RACES = {
+  AMERICAN_INDIAN_ALASKAN_NATIVE: "American Indian or Alaskan Native",
+  ASIAN: "Asian",
```

Comment:
Heads up that we aren't showing the "Asian" racial breakdown for the ND dashboard. We are grouping "Asian" with "Other" because of the small counts. This page seems to be a global constant file, so I'm sure that this will say here but wanted to flag this!

Reply:
Thanks for the heads up, @jjdressel11.  I'm going to leave the constant here for now so that it's available for other states in the future.  Appreciate the flag!